packages/ola/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgbase=ola
+pkgname=ola
+pkgver=0.10.8
+pkgrel=1
+pkgdesc='Open Lighting Architecture for controlling entertainment lighting equipment'
+arch=(x86_64 aarch64)
+url='https://www.openlighting.org'
+license=(LGPL2.1 GPL2)
+depends=()
+makedepends=(avahi cppunit doxygen flake8 libmicrohttpd protobuf)
+source=("${pkgbase}-${pkgver}.tar.gz::https://github.com/OpenLightingProject/${pkgbase}/archive/refs/tags/${pkgver}.tar.gz")
+sha256sums=('45bc101d1ddcc1c6320c063eb25a9345e5820a4233acfa5b29f2dfd99f7452e1')
+
+build() {
+  cd "$srcdir"/${pkgbase}-${pkgver}
+  autoreconf -i
+  ./configure --prefix=/usr
```

Comment:
The [ola-git](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=ola-git) package also enables Python libraries. Is this something we should do too?

Reply:
Good idea! Would it make sense to then have python only as a make dependency? And add `provides=(python-ola)`?

Reply:
Maybe here a split package would make sense too. From what I can see with a quick glance at the OLA website, the Python API is officially supported, but none of the installed programs use it.

Reply:
Relevant discussion on IRC `#archlinux-aur`:

```
<strogon14> If a C library package has optional Python bindings, which can be enabled at compile time, should I put them in the same package or make a split package ('foolib', 'foolib-python')?
<Scimmia> unless there's a compelling reason to split them out, don't
<strogon14> what would be an example of a compelling reason?
<Scimmia> for python bindings, I can't think of any
<Scimmia> but I couldn't rule it out
<Scimmia> for example, systemd libs are split out to prevent issues with install order and post_install scripts not running
<Scimmia> dep cycles can be another
<strogon14> in this case, it's a pure python lib, with no deps other than the standard lib, so the only additional package dependency would be python.
<strogon14> seems like it would be ok to include it.
<Scimmia> I would probably even make the python dep optional
<strogon14> scratch that, it requires google-protobuf, but that can be covered by python-protobuf.
```

Reply:
Adding `provides=(python-ola)` would be a measure to make it future-proof, should the package ever be split, but isn't strictly necessary. But `python` and `python-protobuf` should be in `optdepends` then,

Reply:
[libftdi](https://github.com/archlinux/svntogit-community/blob/packages/libftdi/trunk/PKGBUILD) for example is also packaged like this:
```bash
pkgdesc="A library to talk to FTDI chips, optional python bindings."
optdepends=('python: library bindings')
makedepends=('boost' 'cmake' 'python' 'swig'
```